### PR TITLE
Tokenize a single non-repeatable InputListLOC value

### PR DESCRIPTION
**Pull request by jgreben**
Screen Shot 2019-11-21 at 1 25 39 PM

### CODE REVIEW

File: __tests__/components/editor/property/InputListLOC.test.js [hunk 1]
```diff
@@ -157,25 +157,25 @@ describe('InputListLOC', () => {
...
     const store = createReduxStore(createInitialState())
     const {
-      getByPlaceholderText, queryByText, getByText, findByText, getByDisplayValue,
+      container, getByPlaceholderText, queryByText, getByText, findByText,
     } = renderWithRedux(
       <InputListLOC reduxPath={reduxPath} />, store,
     )
     // The input box is present.
-    expect(getByPlaceholderText('Sound characteristics')).toBeInTheDocument()
+    // expect(getByPlaceholderText('Sound characteristics')).toBeInTheDocument()
```
Comment (justinlittman) [inline __tests__/components/editor/property/InputListLOC.test.js]: Intentionally removed? Cruft?

File: __tests__/components/editor/property/InputListLOC.test.js [hunk 2]
```diff
@@ -343,11 +343,12 @@ describe('InputListLOC', () => {
     // Start typing a new literal
     fireEvent.change(input, { target: { value: 'foo' } })
 
-    expect(await findByText('foo', 'a.dropdown-item')).toBeInTheDocument()
+    // expect(await findByText('foo', 'a.dropdown-item')).toBeInTheDocument()
```
Comment (justinlittman) [inline __tests__/components/editor/property/InputListLOC.test.js]: Intentionally removed? Cruft?

File: __tests__/components/editor/property/InputListLOC.test.js (hunk 1 above)
Comment (jgreben) [reply __tests__/components/editor/property/InputListLOC.test.js]: Cruft

File: __tests__/components/editor/property/InputListLOC.test.js (hunk 2 above)
Comment (jgreben) [reply __tests__/components/editor/property/InputListLOC.test.js]: Intentionally removed

**Pull request merged**